languages/go/concepts/methods/links.json

```
@@ -1 +1,14 @@
-[]
+[
+  {
+    "url": "https://tour.golang.org/methods/1",
+    "description": "methods"
+  },
+  {
+    "url": "https://tour.golang.org/methods/4",
+    "description": "pointers_receivers"
```

Maybe use a user-friendly name here? Something like:

```suggestion
    "description": "Pointer receivers"
```

p.s. I don't know if the above suggestion makes any sense

---

Makes sense.